perf-v2/index.html

```diff
@@ -101,38 +106,46 @@
     <footer>
       <button id="dl-button">Download data as JSON</button>
       <div class="spacer"></div>
-      <div class="small">Powered by <a rel="noopener" href="https://github.com/marketplace/actions/continuous-benchmark">github-action-benchmark</a></div>
+      <div class="small">
+        Powered by
+        <a
+          rel="noopener"
+          href="https://github.com/marketplace/actions/continuous-benchmark"
+          >github-action-benchmark</a
+        >
+      </div>
     </footer>
 
     <script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.2/dist/Chart.min.js"></script>
     <script src="data.js"></script>
     <script id="main-script">
-      'use strict';
-      (function() {
+      "use strict";
+      (function () {
         // Colors from https://github.com/github/linguist/blob/master/lib/linguist/languages.yml
         const toolColors = {
-          cargo: '#dea584',
-          go: '#00add8',
-          benchmarkjs: '#f1e05a',
-          pytest: '#3572a5',
-          googlecpp: '#f34b7d',
-          catch2: '#f34b7d',
-          customBiggerIsBetter: '#38ff38',
-          customSmallerIsBetter: '#ff3838',
-          _: '#333333'
+          cargo: "#dea584",
+          go: "#00add8",
+          benchmarkjs: "#f1e05a",
+          pytest: "#3572a5",
+          googlecpp: "#f34b7d",
+          catch2: "#f34b7d",
+          customBiggerIsBetter: "#38ff38",
+          customSmallerIsBetter: "#ff3838",
+          _: "#333333",
         };
 
         function init() {
           function collectBenchesPerTestCase(entries) {
             const map = new Map();
+            for (const bench of entries[0].benches) {
+              map.set(bench.name, []);
+            }
             for (const entry of entries) {
-              const {commit, date, tool, benches} = entry;
+              const { commit, date, tool, benches } = entry;
```

Comment:
What prettier config are you using for all these changes? It is not consistent with the rest of the code base?

Reply:
and it is causing diff churn...